### ISSUE TITLE
When Botania is or isn't installed, the vacuum tubes bugs visually

### ISSUE THREAD

**Issue by Vapaman** [TeamPneumatic/pnc-repressurized#1057]
### Describe the bug

The outline becomes transparent eg. you can see through the block.
![kuva](https://user-images.githubusercontent.com/77460466/181632883-587cffaf-fbc4-40d5-84f6-a5e3063e5a2a.png)

As a side note, when Botania _is **not** installed_ the rotor of the block disappears when viewed at certain angles or with F1 enabled. The rotor does not disappear when Botania _is installed_
![kuva](https://user-images.githubusercontent.com/77460466/181633455-1e5c4633-6f5e-4d4e-bf60-72b9a6fcb203.png)

### How to reproduce the bug

**The outline problem:**
Have these mods installed:
Botania 1.18.2-434
Curios 1.18.2-5.0.7.1
PNCR 1.18.2-3.3.1-110
Patchouli 1.18.2-71.1

The outline should be transparent when the block is looked at

---------------------------------------
**The disappearing rotor problem:**
PNCR 1.18.2-3.3.1-110
Patchouli 1.18.2-71.1

Press F1 or look at the block at certain angles

### Expected behavior

The block should look normal under all circumstances
![kuva](https://user-images.githubusercontent.com/77460466/181632420-68341a64-0816-496c-920a-c080f4b09989.png)

### Additional details

The disappearing rotor has to do something with the transparent part of the model: if you use spectator while using F1, you can fly inside the pump and see the rotor just fine. I don't know why it is fixed when Botania is installed though.

### Which Minecraft version are you using?

1.18

### Which version of PneumaticCraft: Repressurized are you using?

1.18.2-3.3.1-110

**Comment by desht** [comment]
Nope, not seeing this.  No Botania in my test instance, and the pump is rendering perfectly well, with or without F1.

Are the mods you've listed the *only* mods in your instance?  No Optifine or other "optimization" mods?

**Comment by Vapaman** [comment]
No other mods. Forge version 40.1.68. Using GDLauncher
![kuva](https://user-images.githubusercontent.com/77460466/181994968-4122c750-3332-4b99-819c-eed3d52a544d.png)
![kuva](https://user-images.githubusercontent.com/77460466/181994976-ced33c0b-c8fd-4823-9546-6adce851526d.png)

PC specs in the F3 screen as this might be caused by my computer.
![kuva](https://user-images.githubusercontent.com/77460466/181995004-5f420374-f8fc-4933-99c5-9831e69b8cd9.png)

Fresh install on PolyMC nets the same results.

**Comment by desht** [comment]
Well sorry then, I don't know what to say.  I can't reproduce the problem at all.  I'm also using NVidia (although GTX 16606 on Linux) so unlikely to be a driver problem.

**Comment by Vapaman** [comment]
Maybe you can't reproduce the bug as I'm using Windows 10

**Comment by Vapaman** [comment]
Also happens on 1.19 and 1.16

**Comment by Vapaman** [comment]
Haven't really figured out why this happens, but it seems that it happens with the nether portal block too

**Comment by desht** [comment]
So if it happens with a vanilla block too, then the problem can't be with PNC...

**Comment by Vapaman** [comment]
The outline rendering issue seems to be entirely on botanias side. The issue with nether portals happens with just botania installed. 

The rotor disappearing with F1 still happens with just Pneumaticcraft.

Edit: Also, swithing the graphics setting to Fabulous seems to fix this
